geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TemporalTiledRasterLayer.scala

```diff
@@ -123,6 +123,7 @@ class TemporalTiledRasterLayer(
 
   def tileToLayout(
     layoutDefinition: LayoutDefinition,
+    zoom: Option[Int],
     resampleMethod: ResampleMethod
   ): TiledRasterLayer[SpaceTimeKey] = {
```

Comment:
I covered it here https://github.com/jbouffard/geopyspark/blob/bug-fix/tiledlayer-tile-to-layout/geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterLayer.scala#L95